Missing @xstyled/util dependency in @xstyled/core

## 🐛 Bug Report

`@xstyled/util` is used inside `@xstyled/core` but does not exist inside the `package.json`s dependencies.

https://www.runpkg.com/?@xstyled/core@3.5.1/dist/index.cjs

This is an issue with package managers such as yarn pnp.

```
Module not found: Error: @xstyled/core tried to access @xstyled/util, but it isn't declared in its dependencies; this makes the require call ambiguous and unsound.
```

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -31,6 +31,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@xstyled/system": "^3.5.1"
+    "@xstyled/system": "^3.5.1",
+    "@xstyled/util": "^3.5.1"
   }
 }

